feature_extractor

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+# Convenience script to run features extraction service locally
+
+service_dir="src/main/python/feature-extractor"
+
+hash python3 >/dev/null 2>&1 || { echo "Please install python3" >&2; exit 1; }
+
+if [ -z "$VIRTUAL_ENV" ]; then
+    echo "Looks like you don't use virtual environment for python"
+    echo "Please active it or create a new one"
+    echo "Move information is here: https://virtualenv.pypa.io/en/stable/"
```

Comment:
Small typos:
"Please **activate** it or create a new one"
"**More** information here: https://virtualenv.pypa.io/en/stable/"